File: Fundamentals_of_Software_Engineering/front/lk/src/main/java/org/suai/common/Response.java
package org.suai.common;

public class Response {
    public Role role;
    public String message;

    public Response(String message) {
            role = Role.NONE;
            this.message = message;
    }

    public Response(Role role, String message) {
            this.role = role;
            this.message = message;
    }
}

File: Fundamentals_of_Software_Engineering/front/lk/common/src/commonMain/kotlin/org/suai/common/Response.kt
package org.suai.common

class Response {
    var role: Role
    var message: String

    constructor(message: String) {
        role = Role.NONE
        this.message = message
    }

    constructor(role: Role, message: String) {
        this.role = role
        this.message = message
    }
}